### DISCUSSION TITLE
Need to know these javascript variables names

### DISCUSSION THREAD

**Discussion by intgr111** [tradingview/lightweight-charts#737] [Q&A] [answered]
Hi all,
I use TradingView widget to draw charts inside browser.
Does anybody know JavaScript variables to achieve:
not to show on chart by default:
- volume
- grid
- price line
- last price label
?

**Comment by timocov** [comment]
> I use TradingView widget to draw charts inside browser

This issue tracker/discussions aren't about TradingView's widget. Please post your question on stackoverflow instead.

**Comment by intgr111** [comment]
404
https://github.com/tradingview/charting_library/wiki/Overrides

**Comment by intgr111** [comment, marked as answer]
upd
solved
all naswers are here https://github.com/StellarCN/firefly/blob/master/src/components/TradingViewChart.vue
